fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1404,4 +1404,7 @@ public class Config extends ConfigBase {
      */
     @ConfField(mutable = true)
     public static int default_bucket_num = 10;
+
+    @ConfField(mutable = true)
+    public static boolean enable_experimental_mv = false;
```

Comment:
this name is not good. Suppose that 1 year later, what do users think when they see the name.
By this time, this feature is not a experimental, however the configure is also this name. It will make users feel that this feature is not stable.
So I suggest you should choose another name.

Reply:
Some software, such as Docker, mark certain features as experimental if they are not ready for production use, and remove the flags when the feature is stable.

Reply:
What you mean this is only for debug? I'm OK for this.